docs/DeveloperGuide.md

```diff
@@ -269,6 +269,66 @@ The following activity diagram summarizes what happens when a user executes a ne
   * Pros: Will use more memory, since the HashMap will have to be committed to local storage too.
   * Cons: We must ensure that the any changes/updates to a flashcard will be reflected in the HashMap.
 
+### Favourite/Unfavourite Feature 
+
+#### Implementation
+
+The favourite/unfavourite mechanism is faciliated by `LogicManager` and `ModelManager`.
+A `isFavourite` attribute is stored internally in `Flashcard`, to keep track of whether the flashcard is favourited. When the user favourites a flashcard, `isFavourite` is set to true, and set to false otherwise. 
+ 
+It implements the following operations:
+* `ModelManager#setFlashcard(Flashcard target, Flashcard editedFlashcard)` - Replaces the current flashcard with the favourited/unfavourited flashcard
```

Comment:
I thought setFlashcard was already there in the first place? Or you implement one

Reply:
oh wait ya i will remove it